src/EnergyPlus/HeatBalanceSurfaceManager.hh

```diff
@@ -64,6 +64,8 @@
 
 // EnergyPlus Headers
 #include <EnergyPlus.hh>
+#include <DataSurfaces.hh>
+#include <DataHeatBalance.hh>
```

Comment:
@JasonGlazer Instead of including the full header files for both `DataSurfaces` and `DataHeatBalance` you should instead use forward declarations. Then include these header files in the `HeatBalanceSurfaceManager.cc` file instead. 

Basically within the `namespace EnergyPlus` you will include `class SurfaceData;` and `class ZoneData;`. I will try to find an example to link to.